src/static/exportRoute.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { renderToString, renderToStaticMarkup } from 'react-dom/server'
 import Helmet from 'react-helmet'
-import { ReportChunks } from 'react-universal-component'
+import { clearChunks, flushChunkNames } from 'react-universal-component/server'
```

Comment:
Actually, if you look closely, it says to use the current method for special use cases (like React Static). :)

Reply:
but it works correct this way, maybe you check yourself with your detailed background

Reply:
Yeah, it may appear to function correctly, but if multiple pages are exported at the same time on the same thread (which happens all the time), then the chunks for different pages will collide and mix with each other, resulting in dirty chunkNames. Sometimes they will package more chunks than necessary, or sometimes they are clobbered and missing in production. It's more indirect, but much safer to use the ReportChunks in our situation, especially given the concurrent and distributed way RS builds.